README.md

```diff
@@ -45,10 +44,15 @@ To manage user and session persistence after the original sign in/up we store th
  - [R.swift](https://github.com/mac-cain13/R.swift) for strongly typed and autocompleted assets, segues and more.
 
 
+#### Utilities
+
+We have developed other libraries that can be helpful and you could integrate with the dependency manager of your choice.
+
+- **[PagedLists:](https://github.com/rootstrap/PagedLists)** Custom **UITableView** and **UICollectionView** classes to easily handle pagination.
```

Comment:
maybe RSFontSizes and RSFormView worth a mention too?

Reply:
Sounds good